README.rst

```diff
@@ -5,7 +5,7 @@ EPC SunSpec Demo
 |GitHub|
 
 
-.. |GitHub| image:: https://img.shields.io/github/last-commit/epcpower/sunspec-demo/develop.svg
+.. |GitHub| image:: https://img.shields.io/github/last-commit/epcpower/sunspec-demo
```

Comment:
Perhaps there are more places where we need to change `develop` to `main` after the branch rename.  Specifying the branch makes it report for that rather than random PRs that happen to be the latest commit.  I think that's good?

```suggestion
.. |GitHub| image:: https://img.shields.io/github/last-commit/epcpower/sunspec-demo/main.svg
```